.github/CHANGELOG.md

```diff
@@ -19,403 +19,445 @@
+  [(#327)](https://github.com/XanaduAI/pennylane/pull/327)
 
-### Bug fixes
+<h3>Bug fixes</h3>
 
 * Fixed typo in measurement type error message in qnode.py
-  [#341](https://github.com/XanaduAI/pennylane/pull/341)
+  [(#341)](https://github.com/XanaduAI/pennylane/pull/341)
 
-### Contributors
+<h3>Contributors</h3>
 
 This release contains contributions from (in alphabetical order):
 
 Shahnawaz Ahmed, Ville Bergholm, Aroosa Ijaz, Josh Izaac, Nathan Killoran, Angus Lowe,
 Johannes Jakob Meyer, Maria Schuld, Antal Száva, Roeland Wiersema.
 
----
-
 # Release 0.5.0
 
-### New features since last release
+<h3>New features since last release</h3>
 
 * Adds a new optimizer, `qml.QNGOptimizer`, which optimizes QNodes using
   quantum natural gradient descent. See https://arxiv.org/abs/1909.02108
   for more details.
-  [#295](https://github.com/XanaduAI/pennylane/pull/295)
-  [#311](https://github.com/XanaduAI/pennylane/pull/311)
+  [(#295)](https://github.com/XanaduAI/pennylane/pull/295)
+  [(#311)](https://github.com/XanaduAI/pennylane/pull/311)
 
 * Adds a new QNode method, `QNode.metric_tensor()`,
   which returns the block-diagonal approximation to the Fubini-Study
   metric tensor evaluated on the attached device.
-  [#295](https://github.com/XanaduAI/pennylane/pull/295)
+  [(#295)](https://github.com/XanaduAI/pennylane/pull/295)
 
 * Sampling support: QNodes can now return a specified number of samples
   from a given observable via the top-level `pennylane.sample()` function.
   To support this on plugin devices, there is a new `Device.sample` method.
 
   Calculating gradients of QNodes that involve sampling is not possible.
-  [#256](https://github.com/XanaduAI/pennylane/pull/256)
+  [(#256)](https://github.com/XanaduAI/pennylane/pull/256)
 
 * `default.qubit` has been updated to provide support for sampling.
-  [#256](https://github.com/XanaduAI/pennylane/pull/256)
+  [(#256)](https://github.com/XanaduAI/pennylane/pull/256)
 
 * Added controlled rotation gates to PennyLane operations and `default.qubit` plugin.
-  [#251](https://github.com/XanaduAI/pennylane/pull/251)
+  [(#251)](https://github.com/XanaduAI/pennylane/pull/251)
 
-### Breaking changes
+<h3>Breaking changes</h3>
 
 * The method `Device.supported` was removed, and replaced with the methods
   `Device.supports_observable` and `Device.supports_operation`.
   Both methods can be called with string arguments (`dev.supports_observable('PauliX')`) and
   class arguments (`dev.supports_observable(qml.PauliX)`).
-  [#276](https://github.com/XanaduAI/pennylane/pull/276)
+  [(#276)](https://github.com/XanaduAI/pennylane/pull/276)
 
 * The following CV observables were renamed to comply with the new Operation/Observable
   scheme: `MeanPhoton` to `NumberOperator`, `Homodyne` to `QuadOperator` and `NumberState` to `FockStateProjector`.
-  [#243](https://github.com/XanaduAI/pennylane/pull/254)
+  [(#243)](https://github.com/XanaduAI/pennylane/pull/254)
```

Comment:
Here the #243 seems to be a closed one and #254 might be the one that is wanted to be linked (not sure though)

Reply:
(Just left a suggestion assuming that it is correct)